src/components/Toolbar/index.tsx

```diff
@@ -51,7 +52,7 @@ export default function Toolbar() {
       return () => {};
     }
 
-    const unsubscribe = doc.subscribe((event) => {
+    const unsubscribe = doc.subscribe((event: DocEvent) => {
```

Comment:
Wouldn't the type below be inferred automatically?

Reply:
It becomes type inference, but there was an awkward part in my editor 
(Inferred, but not moved to the inferred part in vscode)